doc/sphinx/proof-engine/ltac.rst

```diff
@@ -1160,6 +1173,388 @@ Printing |Ltac| tactics
 
    This command displays a list of all user-defined tactics, with their arguments.
 
+
+.. _ltac-examples:
+
+Examples of using |Ltac|
+-------------------------
+
+
+About the cardinality of the set of natural numbers
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The first example which shows how to use pattern matching over the
+proof context is a proof of the fact that natural numbers have more
+than two elements. This can be done as follows:
+
+.. coqtop:: in reset
+
+   Lemma card_nat :
+     ~ exists x : nat, exists y : nat, forall z:nat, x = z \/ y = z.
```

Comment:
Indeed, that's a French-speaker usual mistake: in French, “more” is generally understood in the large sense, whereas in English it is understood in the strict sense.

Reply:
Interesting.  Just FYI: It is a "common mistake" rather than a "usual mistake" (usual = "habitually or typically occurring or done; customary").  I would say "that's a common mistake by French-speakers" or "... by francophones".  "greater than or equal" and "[strictly] greater than" would be better than "large sense" and "strict sense" (which are pretty vague).

Of course, it's a quick comment, not something permanent.